scripts/entrypoint.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/ash
-
+yarn delete-invalid-partners
```

Comment:
Is this command be `delete-partners` as `delete-invalid-partners` does not exist for now

Reply:
Sorry about that, change has been effected. Its `delete-partners`